api/src/certification/complementary-certification/application/attach-target-profile-controller.js

```diff
@@ -17,7 +17,7 @@ const attachTargetProfile = async function (request, h, dependencies = { complem
     complementaryCertificationBadgesToAttachDTO: complementaryCertificationBadges,
   });
 
-  if (notifyOrganizations) {
+  if (!!targetProfileId && notifyOrganizations) {
```

Comment:
Pas obligé de caster en boolean avec `!!` ?

Reply:
!! cast en Boolean, mais tu as raison ca equivaut à `Boolean(targetProfileId)`

Reply:
Ha non je voulais dire on peut faire ça direct non ?
```suggestion
if (targetProfileId && notifyOrganizations) {
```